src/Pim/Component/Catalog/Builder/ProductBuilder.php

```diff
@@ -93,6 +93,8 @@ public function createProduct($identifier = null, $familyCode = null)
 
         if (null !== $identifier) {
             $product->setIdentifier($identifier);
+            $identifierAttribute = $this->attributeRepository->getIdentifier();
```

Comment:
LoadProductValuesListener is a Doctrine subscriber, not compatible with our event.
And as we should always use the builder to create new products, it should not be a problem.